src/algebra/order/to_interval_mod.lean

```diff
@@ -732,4 +732,26 @@ end
 lemma to_Ico_mod_zero_one (x : α) : to_Ico_mod (0 : α) zero_lt_one x = int.fract x :=
 by simp [to_Ico_mod_eq_add_fract_mul]
 
+lemma Union_Ioc_int_cast : (⋃ (n : ℤ), set.Ioc (n:α) (n + 1)) = set.univ :=
+begin
+  refine set.eq_univ_iff_forall.mpr (λ x, set.mem_Union.mpr _),
+  have := sub_to_Ioc_div_zsmul_mem_Ioc 0 zero_lt_one x,
+  rw int.smul_one_eq_coe at this,
+  exact ⟨to_Ioc_div 0 zero_lt_one x, ⟨sub_pos.mp this.1, by linarith [this.2]⟩⟩,
+end
+
+lemma pairwise_disjoint_Ioc_int_cast {𝕜 : Type*} [linear_ordered_field 𝕜] :
+  pairwise (disjoint on (λ (m:ℤ), set.Ioc (m:𝕜) (m + 1))) :=
+begin
+  intros m n h,
+  rw [function.on_fun, disjoint_iff, set.inf_eq_inter, set.bot_eq_empty,
+    set.eq_empty_iff_forall_not_mem],
+  contrapose! h,
+  obtain ⟨u, hu1, hu2⟩ := h,
+  have i1 := hu1.1.trans_le hu2.2,
+  have i2 := hu2.1.trans_le hu1.2,
+  rw [←int.cast_one, ←int.cast_add, int.cast_lt, int.lt_add_one_iff] at i1 i2,
+  exact le_antisymm i1 i2,
+end
```

Comment:
This file is about `to_Ico_mod`, but it's not the case of these lemmas. Could you move them to a better place (or add a porting note if the file is already ported)? Also, the assumptions are too strong, you don't need a linear ordered field here. `linear_ordered_ring`?

Reply:
@YaelDillies, is this the preferred spelling for "these sets are pairwise disjoint and union to `univ`"? Another approach that comes to mind is to write the equiv between `𝕜` and `ℤ \times set.Ioc 0 b` or `𝕜`  and `\sigma z : ℤ, set.Ioc (a + z • b)  (a + (z + 1) • b)

Reply:
This is the formulation that `has_sum_integral_Union` expects.

Reply:
This looks fine to me.